test/microapps-ui-xcm/microapps-ui-main.check.pools.test.ts

```diff
@@ -0,0 +1,230 @@
+import { ApiContext } from "../../utils/Framework/XcmHelper";
+import XcmNetworks from "../../utils/Framework/XcmNetworks";
+import { connectVertical } from "@acala-network/chopsticks";
+import { devTestingPairs } from "../../utils/setup";
+import StashServiceMockSingleton from "../../utils/stashServiceMockSingleton";
+import { AssetId } from "../../utils/ChainSpecs";
+import { getEnvironmentRequiredVars } from "../../utils/utils";
+import { AssetWallet, User } from "../../utils/User";
+import {
+  connectWallet,
+  setupPage,
+} from "../../utils/frontend/microapps-utils/Handlers";
+import "jest-extended";
+
+jest.setTimeout(FIVE_MIN);
+jest.spyOn(console, "log").mockImplementation(jest.fn());
+let driver: WebDriver;
+let testUser1: User;
+
+const acc_name = "acc_automation";
+const userAddress = "5CfLmpjCJu41g3cpZVoiH7MSrSppgVVVC3xq23iy9dZrW2HR";
+
+describe("Microapps UI liq pools tests", () => {
+  let kusama: ApiContext;
+  let mangata: ApiContext;
+  let alice: KeyringPair;
+  let sdk: MangataInstance;
+  let api: ApiPromise;
+
+  beforeAll(async () => {
+    kusama = await XcmNetworks.kusama({ localPort: 9944 });
+    mangata = await XcmNetworks.mangata({ localPort: 9946 });
+    await connectVertical(kusama.chain, mangata.chain);
+    alice = devTestingPairs().alice;
+    StashServiceMockSingleton.getInstance().startMock();
+
+    try {
+      getApi();
+    } catch (e) {
+      await initApi();
+    }
+
+    sdk = await getMangataInstance();
+    api = getApi();
+
+    await mangata.dev.setStorage({
+      Tokens: {
+        Accounts: [
+          [[userAddress, { token: 4 }], { free: 10 * 1e12 }],
+          [[userAddress, { token: 7 }], { free: 10 * 1e10 }],
+          [
+            [userAddress, { token: 26 }],
+            { free: AssetId.Mgx.unit.mul(BN_THOUSAND).toString() },
+          ],
+          [
+            [userAddress, { token: 0 }],
+            { free: AssetId.Mgx.unit.mul(BN_TEN_THOUSAND).toString() },
+          ],
+          [[alice.address, { token: 4 }], { free: 10 * 1e12 }],
+          [
+            [alice.address, { token: 0 }],
+            { free: AssetId.Mgx.unit.mul(BN_THOUSAND).toString() },
+          ],
+        ],
+      },
+      Sudo: {
+        Key: userAddress,
+      },
+    });
+
+    driver = await DriverBuilder.getInstance();
+    await importPolkadotExtension(driver);
+
+    const keyring = new Keyring({ type: "sr25519" });
+    const node = new Node(getEnvironmentRequiredVars().chainUri);
+    await node.connect();
+
+    testUser1 = new User(keyring);
+    testUser1.addFromMnemonic(
+      keyring,
+      getEnvironmentRequiredVars().mnemonicPolkadot,
+    );
+
+    testUser1.addAsset(KSM_ASSET_ID);
+    testUser1.addAsset(MGA_ASSET_ID);
+    await testUser1.refreshAmounts(AssetWallet.BEFORE);
+
+    await setupPage(driver);
+    await connectWallet(driver, "Polkadot", acc_name);
+  });
+
+  it("All promoted pools exist in app", async () => {
+    const promotedPoolsRewards =
```

Comment:
these are pools, not rewards. plz rename

Reply:
I merged it with the next var